Packages/StatusKit/Sources/StatusKit/Row/Subviews/StatusRowCardView.swift

```diff
@@ -173,3 +156,54 @@ public struct StatusRowCardView: View {
     }
   }
 }
+
+struct DefaultPreviewImage: View {
+  let url: URL
+  let originalWidth: CGFloat
+  let originalHeight: CGFloat
+
+  var body: some View {
+    _Layout(originalWidth: originalWidth, originalHeight: originalHeight) {
+      LazyResizableImage(url: url) { state, _ in
+        Rectangle().fill(Color.gray)
+          .overlay {
+            if let image = state.image {
+              image.resizable().scaledToFill()
+            }
+          }
+      }
+      .accessibilityHidden(true) // This image is decorative
+      .clipped()
+    }
+  }
+
+  private struct _Layout: Layout {
```

Comment:
Very interesting! I've never used the Layout API! I need to read about it.

Reply:
We can further stabilize the timeline layout with it. 😁

Reply:
Yes, I need to read about it! I feel like this could help for more stuff!